### DISCUSSION TITLE
Cannot update child in the Question set

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#464] [Bugs] [answered]
Hello All,

I've created a **Question set** and which is currently live.

Following are the steps that I've followed to update the Live Question set:
1. Made changes in the Child section and was able to successfully sent it for review.
2. Tried to review the Question set from the Reviewer's workspace, not able to see the updated changes made in the child section.

I tried to check the network panel also, there is no API getting failed.

**I need to understand how we can update the child section in Question set?**

cc: @rajeevsathish @sharathkashyap @vinukumar-vs @rhwarrier @reshmi-nair @Krishnaj20

**Comment by Krishnaj20** [comment, marked as answer]
@nidhikachhawa24 - since this is related to workspace request you to post this query in coKreat forums. Link: https://github.com/orgs/Sunbird-coKreat/discussions

**Reply by nidhikachhawa24** [reply]
@Krishnaj20 
Whom should I tag there?

**Reply by Krishnaj20** [reply]
pls raise the discussion thread. coKreat community will check and respond.
